governance/polls/Vault Compensation Poll 3

```diff
@@ -0,0 +1,54 @@
+
+title: Vault Compensation Poll
+summary: Signal your support for vault compensation
+discussion link: https://forum.makerdao.com/t/vault-compensation-plan-v2/3584
+vote type: Ranked Choice Voting
+options: 
+   0: Abstain
+   1: 24.67% (theoretical maximum auction yield) - up to 25,925 MKR and 50,000 DAI 
+   2: 21% - up to 21,627 MKR and 50,000 DAI
+   3: 18% - up to 18,216 MKR and 50,000 DAI
+   4: 15% - up to 14,871 MKR and 50,000 DAI
+   5: 12% - up to 11,691 MKR and 50,000 DAI
+   6: 9% - up to 5,592 MKR and 50,000 DAI
+   7: 6% - up to 2,748 MKR and 50,000 DAI
+   8: 3% - up to 917 MKR and 50,000 DAI
+   9: 0% (oppose vault compensation) - 0 MKR and 0 DAI
+   10: Oppose current compensation plan (support vault compensation) 0 MKR and 0 DAI
+---
+# Vault Compensation Plan Approval Poll
+
+The Maker Foundation Interim Governance Facilitator has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the community, which the community can use to signal support for compensating vaults affected by liquidations during the March 12 market crash.
+
+## Review
+
+The Vault Compensation Plan was discussed in the [Forums](https://forum.makerdao.com/t/vault-compensation-plan-v2/3584) and presented on the [August 13 Maker Governance and Risk Call](https://forum.makerdao.com/t/agenda-discussion-scientific-governance-and-risk-105-thursday-august-13-4-00-pm-utc/3570). Voters are encouraged to review the plan document and discussion to inform their position before voting.
+
+This poll uses ranked choice voting. Details about ranked choice voting can be found [here](https://forum.makerdao.com/t/signal-request-add-ranked-choice-voting-as-an-option-for-governance-polls/1274). The community is encouraged to select multiple options in their order of preference, while withholding their vote for any options they consider unacceptable. 
+
+Claim amounts will be calculated based upon the compensation percentage decided in this poll, using the “tab change in relation to collateral change” calculation method described in the plan document, paid out in MKR tokens at a rate of 1 MKR per 250 DAI. Individual vault compensation will be determined by the formulae below:
+
+`MKR compensation amount = (collateral deficit percent * collateral value in DAI) / 250`
```

Comment:
Can you get MakerMan to confirm this is correct? I'd just like more eyes on it before putting it in the poll.

Reply:
Though I can't see anything obviously incorrect here.